src/y-webrtc.js

```diff
@@ -13,7 +13,9 @@ import * as math from 'lib0/math.js'
 import { createMutex } from 'lib0/mutex.js'
 
 import * as Y from 'yjs' // eslint-disable-line
-import Peer from 'simple-peer/simplepeer.min.js'
+
+// import Peer from 'simple-peer/simplepeer.min.js'
+import Peer from './SimplePeerExtended'
```

Comment:
This is interesting. So you implemented a layer around simple-peer that handles this issue.

Would it be possible that you publish a separate package that we can include as a polyfill for the default implementation? This is already done in y-websocket where we define the `WebSocket` as an argument.

```js
new WebsocketProvider(URL, room, { WebSocket: MyCustomWebsocketPolyfill })
```

We could do something similar to y-webrtc without breaking the existing API. 

```js
new WebrtcProvider(room, yjs, { SimplePeer: SimplePeerExtended })
```

I'd prefer that approach because it allows us to test out your implementation before breaking everyone else's existing deployments.

Reply:
i can publish a module and put something up on github, but it seems like you'd need move SimplePeer to a peerDependency to avoid duplicating code, otherwise your library is pulling in Peer while your user is also pulling in a modified peer.

Also, it looks like ```WebrtcConn``` is what's using the ```Peer``` and that's being called from ```SignalingConn```...etc...

I can't figure out how you'd thread that option into the proper slot because it call comes from a global ```import Peer from 'simple-peer/simplepeer.min.js'```